packages/components/src/components/Text/Text.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { View } from 'react-native'
+import React from 'react'
+
+import { Text, TextProps } from './Text'
+import { generateDocs } from '../../utils/storybook'
+
+const meta: Meta<TextProps> = {
+  title: 'Text',
+  component: Text,
+  decorators: [
+    (Story) => (
+      <View
+        style={{
+          flex: 1,
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}>
+        <Story />
+      </View>
+    ),
+  ],
+  parameters: {
+    docs: generateDocs({
+      name: 'Text',
+      // docUrl:
+      //   'https://department-of-veterans-affairs.github.io/va-mobile-app/design/Components/Alerts%20and%20Progress/Text',
```

Comment:
Not sure how much the stories were changing, but highlighting this needs updating if that storybook ticket was rolled into this PR.

Reply:
Will keep the storybook ticket open until Jessica is done w/ the docs